src/seedu/addressbook/parser/Parser.java

```diff
@@ -245,10 +253,25 @@ private Command prepareFind(String args) {
         }
 
         // keywords delimited by whitespace
-        final String[] keywords = matcher.group("keywords").split("\\s+");
-        final Set<String> keywordSet = new HashSet<>(Arrays.asList(keywords));
+        final Set<String> keywordSet = getKeywordSet(matcher);
         return new FindCommand(keywordSet);
     }
 
+    private Command prepareFindTag(String args) {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.